packages/terra-menu/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-menu",
   "main": "lib/Menu.js",
-  "version": "6.78.0",
+  "version": "6.79.0",
```

Comment:
Correct me if I am wrong, but I had an impression that fixes are patches?

Reply:
Yes both of these you pointed out should be patch versions, good catch!

Reply:
[59e4ab1](https://github.com/cerner/terra-framework/pull/1832/commits/59e4ab1088afe400e2915ca480cb7a13ac1a3964)